app/src/testWebkit/java/org/mozilla/focus/telemetry/FirebaseEventTest.java

```diff
@@ -88,51 +88,18 @@ public void paramSizeTooLarge_shouldThrow() {
         firebaseEvent.param("", "");
     }
 
-    @Test
-    public void paramSizeValid_shouldBeSafe() {
-        final FirebaseEvent firebaseEvent = generateValidFirebaseEvent();
-        final Bundle extras = mock(Bundle.class);
-        firebaseEvent.setParam(extras);
-
-        when(extras.size()).thenReturn(FirebaseEvent.MAX_PARAM_SIZE);
-        try {
-            firebaseEvent.param("", "");
-        } catch (Exception e) {
-            fail("Should not have thrown any exception");
-        }
-    }
 
     @Test(expected = IllegalArgumentException.class)
     public void paramKeyLengthTooLarge_shouldThrow() {
         final FirebaseEvent firebaseEvent = generateValidFirebaseEvent();
         firebaseEvent.param(INVALID_PARAM_NAME, "");
     }
 
-    @Test
-    public void paramKeyValid_shouldBeSafe() {
-        final FirebaseEvent firebaseEvent = generateValidFirebaseEvent();
-        try {
-            firebaseEvent.param(VALID_PARAM_NAME, "");
-        } catch (Exception e) {
-            fail("Should not have thrown any exception");
-        }
-    }
-
     @Test(expected = IllegalArgumentException.class)
     public void paramValueLengthTooLarge_shouldThrow() {
         final FirebaseEvent firebaseEvent = generateValidFirebaseEvent();
         firebaseEvent.param("", INVALID_PARAM_VALUE);
     }
 
-    @Test
-    public void paramValueLengthValid_shouldBeSafe() {
-        final FirebaseEvent firebaseEvent = generateValidFirebaseEvent();
```

Comment:
I somewhat feel like keeping this is capturing more errors for us :P but it's up to you!

Reply:
agree